src/shared/services/bolt/bolt.js

```diff
@@ -22,181 +22,102 @@ import { v4 } from 'uuid'
 import { v1 as neo4j } from 'neo4j-driver-alias'
 import { stringifyMod } from 'services/utils'
 import * as mappings from './boltMappings'
-import { BoltConnectionError, createErrorObject } from '../exceptions'
+import * as boltConnection from './boltConnection'
+import { runCypherMessage, cancelTransactionMessage, CYPHER_ERROR_MESSAGE, CYPHER_RESPONSE_MESSAGE, POST_CANCEL_TRANSACTION_MESSAGE } from './boltWorkerMessages'
 
-let _drivers = null
-let _useRoutingConfig = false
-let _routingAvailable = false
-const runningQueryRegister = {}
+let connectionProperties = null
+let boltWorkerRegister = {}
+let cancellationRegister = {}
+let enableWebWorkers = true
```

Comment:
Maybe we should start off by have this as a "experimental" setting in the config pane?
wdyt?